rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -31,6 +31,12 @@
 
 inline std::string sstr(const QString& _in) { return _in.toStdString(); }
 
+template<typename T>
+static QSet<T> list_to_set(const QList<T>& list)
+{
+	return QSet(list.begin(), list.end());
```

Comment:
QSet<T> ?

Reply:
I’ll test that, weird though that it was building fine locally.